src/node/chainstate.cpp

```diff
@@ -18,6 +17,7 @@ std::optional<ChainstateLoadingError> LoadChainstate(bool fReset,
                                                      int64_t nBlockTreeDBCache,
                                                      int64_t nCoinDBCache,
                                                      int64_t nCoinCacheUsage,
+                                                     std::optional<std::function<bool()>> shutdown_requested,
```

Comment:
In commit "node/chainstate: Decouple from ShutdownRequested" (258ce97cf72043ffa69d2b77a4438df723c734a6)

Function objects can be null so here and other places should replace `optional<function<X>>` with `function<X>` so there is one null value instead of two